### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by Mandar-Shinde**
updated readme with text formatting and banner image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
+<div align="center">
+  <img src="https://github.com/twitter/finagle/blob/develop/doc/src/sphinx/_static/logo_medium.png"><br><br>
+</div>
+
+
 # Finagle
+This project is used in production at Twitter (and many other organizations),and is being actively developed and maintained.
```
Comment (mosesn) [inline README.md]: s/,and/, and

**Comment by yufangong** [comment]
@Mandar-Shinde Do you mind also updating two other repos so we could keep these consistent?
They are [Finatra](https://github.com/twitter/finatra), and [Dodo](https://github.com/twitter/dodo), thank you.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 
 [![Build status](https://travis-ci.org/twitter/finagle.svg?branch=develop)](https://travis-ci.org/twitter/finagle)
 [![Codecov](https://codecov.io/gh/twitter/finagle/branch/develop/graph/badge.svg)](https://codecov.io/gh/twitter/finagle)
 [![Project status](https://img.shields.io/badge/status-active-brightgreen.svg)](#status)
 [![Gitter](https://badges.gitter.im/twitter/finagle.svg)](https://gitter.im/twitter/finagle?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.twitter/finagle-core_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.twitter/finagle-core_2.12)
 
-## Status
```
Comment (yufangong) [inline README.md]: Let's keep this Status subtitle.

Comment (Mandar-Shinde) [reply README.md]: @yufangong sure, will do.

File: README.md [hunk 3]
```diff
@@ -7,11 +12,8 @@
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.twitter/finagle-core_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.twitter/finagle-core_2.12)
 
 ## Status
+This project is used in production at Twitter (and many other organizations),and is being actively developed and maintained.
```
Comment (yufangong) [inline README.md]: Please add another new line to be consistent with other subtitles.

**Comment by kevinoliver** [comment]
Thanks for the patch. This got merged in https://github.com/twitter/finagle/commit/54c6f2728c2ed6e091909b447e362f04ab6fe81e

**Pull request closed without merging**